src/sif/autocost.cc

```diff
@@ -270,7 +270,7 @@ class AutoCost : public DynamicCost {
    * estimate is less than the least possible time along roads.
    */
   virtual float AStarCostFactor() const {
-    return speedfactor_[kMaxSpeedKph];
+    return speedfactor_[kMaxAssumedSpeed];
```

Comment:
This breaks the assumptions of the heuristic. But changing it to use the new `kMaxSpeedKph` would change existing routes I believe...